app/__init__.py

```diff
@@ -72,18 +66,9 @@
 
 app.config.from_object(config)
 
-CORS(
-    app,
-    allow_headers=["X-Requested-With"],
-    allow_origin=app.config["ALLOW_ORIGIN"],
-)
```

Comment:
Is this not needed anymore? `ALLOW_ORIGIN` is not used anywhere else.

Reply:
indeed, all the CORS things are handled by traefik now.

Reply:
indeed, all the CORS things are handled by traefik now.